scripts/release.js

```diff
@@ -193,4 +204,11 @@ if (process.platform == "linux") {
   fs.copySync(extensionsSourceDirectory, extensionsDestDirectory, {deference: true});
   fs.copySync(textmateServiceSourceDirectory, textmateServiceDestDirectory, {deference: true});
   fs.removeSync(path.join(platformReleaseDirectory, "setup.json"));
+
+  // Now that we've copied set the app icon up correctly.
+  // rcedit is imported here as its a big import, only needed for windows.
+  // The image path here does not need to be in the release dir, as its just to do the embed.
+  const rcedit = require("rcedit");
```

Comment:
Sweet! Thanks for finding this and hooking it up to our release script, @CrossR 👍